Bmx.Idp.Okta/OktaApi.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Bmx.Idp.Okta.Models;
+
+namespace Bmx.Idp.Okta {
+	internal class OktaApi : IOktaApi {
+		private readonly JsonSerializerOptions _serializeOptions =
+			new JsonSerializerOptions {PropertyNamingPolicy = JsonNamingPolicy.CamelCase,};
```

Comment:
nit: are you using [D2LTeamSettings.vssettings](https://github.com/Brightspace/vssettings)? This doesn't seem like it..

Reply:
No, I got a modified version of the .editorconfig from the monorepo. I removed some things from there to better align it with the Roslyn defaults (what Microsoft recommends).